resources/function_help/json/concave_hull

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "concave_hull",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns a possibly concave polygon that contains all the points in the geometry",
+  "arguments": [{
+    "arg": "geometry",
+    "description": "a geometry"
+  }, {
+    "arg": "targetPercent",
+    "description": "the percentage of area of the convex hull the solution tries to approach. A target_percent of 1 gives the same result as the convex hull. A target_percent between 0 and 0.99 produces a result that should have a smaller area than the convex hull."
+  }, {
+    "arg": "allowHoles",
+    "optional": true,
+    "default": false,
+    "description": "Is holes allowed into the concave polygon?"
```

Comment:
Should this be "**Are** holes ...?
Or, following [some pattern](https://docs.qgis.org/testing/en/docs/user_manual/expressions/functions_list.html#nodes-to-points) we already have (a bit more verbose I know)

```suggestion
    "description": "optional argument specifying whether to allow holes within the output geometry. Defaults to FALSE, set to TRUE to avoid including holes in the output collection."
```